src/sqlfluff/config.ini

```diff
@@ -1,4 +1,4 @@
 # Config file for SQLFluff, mostly for version info for now
 [sqlfluff]
-version=0.7.1
-stable_version=0.7.1
```

Comment:
Annoyingly this also needs to be done in `plugins/sqlfluff-templator-dbt`. Both get released at once and pypi will complain about releasing an already released version again. @alanmcruickshank did try to make them dependent on the same config file but couldn't get it working :-( Eventually this might be pulled out to another repo so they can be released independently.